test/validixir_test.exs

```diff
@@ -6,6 +6,12 @@ defmodule ValidixirTest do
   alias Validixir.Failure
   alias Validixir.Error
 
+  test "validate/2 with empty validation always succeeds" do
+    result = Validixir.validate(fn -> :ok end, [])
+
+    assert result == :ok
```

Comment:
the result should be `{:ok, :ok}`, since the validation result gets wrapped in a success (which is an ok-Tuple)